crabs/detection_tracking/train_model.py

```diff
@@ -119,14 +120,17 @@ def train_model(self):
             max_epochs=self.config["num_epochs"],
             accelerator=self.accelerator,
             logger=mlf_logger,
+            fast_dev_run=self.args.fast_dev_run,
+            limit_train_batches=self.args.limit_train_batches,
```

Comment:
```suggestion
            fast_dev_run=self.debug_fast_dev_run,
            limit_train_batches=self.debug_limit_train_batches,
```

Reply:
Question: do we want these parameters in the config instead? I personally like them as CLI arguments but I'm not sure we have a clear distinction for what goes in config and what doesn't... 

The big advantage of putting it in the config is that all the config is dumped into an MLflow log. Could we also dump the CLI arguments to an MLflow log? I think this would be very useful. We can discuss this offline or open an issue.

Reply:
I think technically, things like hyper parameters in config, things we need to change quickly and many times in cli. Like debugging so we don't keep changing the config files for example from True to False too much. I don't know really, I think it's preference as well.

Reply:
I think we keep as it is for now, we can always move things around especially if we see thing growing too long